raiden/tests/utils/factories.py

```diff
@@ -1068,18 +1066,24 @@ def get_hop(self, channel_index: int) -> HopState:
     def get_hops(self, *args) -> List[HopState]:
         return [self.get_hop(index) for index in (args or range(len(self.channels)))]
 
-    def get_route(self, channel_index: int) -> RouteState:
+    def get_route(self, channel_index: int, estimated_fee: FeeAmount = 0) -> RouteState:
```

Comment:
Shouldn't Default here be `FeeAmount(0)` ?

Reply:
The tests don't enforce `mypy`.

Reply:
Yes I know but that does not mean we should follow it :)